dsl/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/expression/PowerDslTest.kt

```diff
@@ -0,0 +1,73 @@
+package com.linecorp.kotlinjdsl.dsl.jpql.expression
+
+import com.linecorp.kotlinjdsl.dsl.jpql.entity.employee.Employee
+import com.linecorp.kotlinjdsl.dsl.jpql.queryPart
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import com.linecorp.kotlinjdsl.querymodel.jpql.path.Paths
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+class PowerDslTest : WithAssertions {
+    private val baseExpression = Paths.path(Employee::age)
+    private val exponentExpression = Expressions.value(2)
+    private val exponentPrimitive = 2
```

Comment:
This seems to be done to make the semantics of the variables clearer, but it would be nice to have a consistent style with other existing test code (e.g. ModDslTest.kt)
```suggestion
    private val intExpression1 = Paths.path(Employee::age)
    private val intExpression2 = Expressions.value(2)

    private val int1 = 2
```